package.json

```diff
@@ -11,45 +11,63 @@
     "serve": "gatsby serve",
     "clean": "gatsby clean",
     "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1",
-    "type-check": "tsc --noEmit"
+    "type-check": "tsc --noEmit",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
   },
   "dependencies": {
+    "@debtcollective/dc-popup-component": "^0.10.2",
```

Comment:
Is this required?

Reply:
I thought you've added and I keep it while I rebase u.u
_My bad thanks for checking it_